### PR TITLE
Feature/zip code

### PR DESCRIPTION
**Pull request by jawad12345A**
Developers can now enable support for zip code

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -5,7 +5,7 @@ import 'package:flutter_google_places/flutter_google_places.dart';
 import 'package:flutter/material.dart';
 import 'dart:math';
 
-const kGoogleApiKey = "API_KEY";
+const kGoogleApiKey = "API_KEy";
```
Comment (bobagold) [inline example/lib/main.dart:8]: is it intended change? looks like a typo

File: example/lib/main.dart [hunk 2]
```diff
@@ -111,6 +111,8 @@ class _MyAppState extends State<MyApp> {
       context: context,
       apiKey: kGoogleApiKey,
       onError: onError,
+      enabelZipCode: true
+      ,
```
Comment (bobagold) [inline example/lib/main.dart:115]: can you please run dartfmt?

File: example/ios/Flutter/flutter_export_environment.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```
Comment (bobagold) [inline example/ios/Flutter/flutter_export_environment.sh:2]: the file should not be here

File: lib/src/flutter_google_places.dart [hunk 4]
```diff
@@ -3,10 +3,16 @@ library flutter_google_places.src;
...
 import 'package:flutter/material.dart';
+import 'package:geocoder/geocoder.dart';
 import 'package:google_maps_webservice/places.dart';
 import 'package:http/http.dart';
 import 'package:rxdart/rxdart.dart';
 
+
+
+
+var globalEnableZipCode = false;
```
Comment (bobagold) [inline lib/src/flutter_google_places.dart:14]: it does not have to be global+public, consider making it private by renaming it to `_globalEnableZipCode` or making it non-global

**Comment by juliansteenbakker** [comment]
I am closing this since the requested changes haven't been fixed.

**Pull request closed without merging**